platform_umbrella/apps/home_base_web/lib/home_base_web/controllers/installation_script_controller.ex

```diff
@@ -0,0 +1,18 @@
+defmodule HomeBaseWeb.InstallScriptController do
+  use HomeBaseWeb, :controller
+  use HomeBaseWeb, :verified_routes
+
+  require EEx
+
+  EEx.function_from_file(:defp, :render_install_script, "priv/raw_files/install_script.sh", [:spec_url])
+
+  action_fallback HomeBaseWeb.FallbackController
+
+  def show(conn, %{"installation_id" => install_id}) do
+    script = render_install_script("https://example.com/my-install-spec-url/#{install_id}")
```

Comment:
I still need to get the correct URL here. This works as a POC to show that the template renders and everything.